src/Plugin.php

```diff
@@ -45,6 +45,11 @@ public static function init() {
 
     // Allow SVG files in media library.
     add_filter('upload_mimes', __CLASS__ . '::upload_mime_types');
+    // Register filters to remove the width and height attributes for SVG files.
```

Comment:
1. Still missing a newline to previous chunk (see chunks before and after this for examples).

1. The comment is too technical. It should be a higher-level description here, such as _"Remove size attributes from SVG images, so they scale correctly."_